tldr.py

```diff
@@ -444,7 +448,7 @@ def create_parser() -> ArgumentParser:
                         help="List all available commands for operating system")
 
     parser.add_argument('-s', '--source',
-                        default=PAGES_SOURCE_LOCATION,
+                        default=None,
```

Comment:
This doesn't change the default behaviour. It moves it to the  `get_pages_source_locations()` function.